content/vendor/guides/internal-starter-repo.md

```diff
@@ -0,0 +1,87 @@
+---
+date: "2020-02-20T00:00:00Z"
+lastmod: "2020-02-20T00:00:00Z"
+title: "Internal Starter Repo"
+weight: "1007"
+---
+
+The KOTS Internal Starter Repo guide is an advanced workflow that allows you to use a similar workflow to GitHub Actions but for your own internal on-prem repo and CI/CD systems like GitLab, CircleCI, TravisCI, Jenkins, etc.
+
+This guide will assume you've already completed the [Automating Your Workflow](/vendor/guides/quickstart/#automating-your-workflow) and set up a local git repo to manage your application.
+
+It is broken into four sections:
+
+- [Starter App](#starter-app)
+- [Set Environment Variables](#set-environment-variables)
+- [Plain Kubernetes](#plain-kubernetes)
+- [Helm Chart](#helm-chart)
+
+## Starter App
+
+Clone `kotsapps` repo.
+```shell
+git clone git@github.com:replicatedhq/kotsapps.git
+```
+
+Create a new repo `kots-app` in your internal git org and clone this internal repo
+```shell
+git clone git@git.company.com:team-name/kots-app.git
+```
+
+There are 2 dirs in `kotsapps/starter-repo/`. We'll talk about these in detail in the next few sections.
+```shell
+$ ls kotsapps/starter-repo/
+
+k8s       helm-chart
+```
+
+But first lets set some env vars.
+
+## Set Environment Variables
+
+Set 2 environment variables in your CI/CD configuration.
+
+- Set `REPLICATED_APP` with the App Slug.
+- Set `REPLICATED_API_TOKEN` with the API token.
+
+## Plain Kubernetes
+
+If you just have a simple kubernetes app without helm charts, copy the contents of the `k8s` dir to your new `kots-app` repo.
+```shell
+cp -R kotsapps/starter-repo/k8s/* kots-app/
+```
+
+Commit and push your `kots-app` repo from the `master` branch.
+
+```shell
+$ cd kots-app
+$ git add .
+$ git commit -m "New Kots App"
+$ git push
```

Comment:
Let's not use the `$` prefix for multiline command blocks, per the style guide: https://github.com/replicatedhq/help-center/blob/master/CONTRIBUTING.md#multi-line-code-snippets-should-not-have-the-bash-prompt